nipype/interfaces/ants/preprocess.py

```diff
@@ -0,0 +1,358 @@
+# -*- coding: utf-8 -*-
+"""The ants module provides basic functions for interfacing with ants
+    functions.
+"""
+import csv
+import math
+import os
+
+from ..base import (BaseInterface, BaseInterfaceInputSpec, TraitedSpec, File,
+                    traits, isdefined, Str)
+from .base import ANTSCommand, ANTSCommandInputSpec
+from ...utils.filemanip import split_filename
+
+
+def _extant(field):
+    return (field is not None) and isdefined(field)
+
+class AntsMotionCorrStatsInputSpec(ANTSCommandInputSpec):
+    ''' Input spec for the antsMotionCorrStats command '''
+    mask = File(argstr="-x %s", mandatory=True,
+                desc="compute displacements within specified mask.")
+    moco = File(
+        argstr="-m %s", mandatory=True,
+        desc="motion correction parameter file to calculate statistics on"
+    )
+    output = File(argstr="-o %s", hash_files=False, genfile=True,
+                  desc="csv file to output calculated statistics into")
+    framewise = traits.Bool(argstr="-f %d",
+                            desc="do framwise summarywise stats")
+    output_spatial_map = File(argstr='-s %s', hash_files=False,
+                              desc="File to output displacement magnitude to.")
+
+class AntsMotionCorrStatsOutputSpec(TraitedSpec):
+    ''' Output spec for the antsMotionCorrStats command '''
+    spatial_map = File(desc="output image of displacement magnitude", exists=True)
+    output = File(desc="CSV file containg motion correction statistics", exists=True)
+
+class AntsMotionCorrStats(ANTSCommand):
```

Comment:
See comment below. I'd suggest `MotionCorrectionStats` instead of `AntsMotionCorrStats`